api.md

```diff
@@ -3406,138 +3312,143 @@ SELECT pg_size_pretty(after_compression_total_bytes) as total
 total | 48 kB
 
 ```
-## hypertable_relation_size() [](hypertable_relation_size)
+---
+
+## hypertable_detailed_size() :community_function: [](hypertable_detailed_size)
 
-Get relation size of hypertable like `pg_relation_size(hypertable)`.
+Get size of hypertable like `pg_relation_size(hypertable)`.
+If this is a distributed hypertable, the function returns size
+information per node. All sizes are reported in bytes.
 
-#### Required Arguments [](hypertable_relation_size-required-arguments)
+#### Required Arguments [](hypertable_detailed_size-required-arguments)
 
 |Name|Description|
 |---|---|
 | `main_table` | Identifier of hypertable to get relation size for.|
 
-#### Returns [](hypertable_relation_size-returns)
+#### Returns [](hypertable_detailed_size-returns)
 |Column|Description|
 |---|---|
 |table_bytes|Disk space used by main_table (like pg_relation_size(main_table))|
 |index_bytes|Disk space used by indexes|
 |toast_bytes|Disk space of toast tables|
 |total_bytes|Total disk space used by the specified table, including all indexes and TOAST data|
+|node_name| Node for which size is reported, applicable only to distributed hypertables|
 
-#### Sample Usage [](hypertable_relation_size-examples)
-```sql
-SELECT * FROM hypertable_relation_size('conditions');
-```
-or, to reduce the output, a common use is:
+#### Sample Usage [](hypertable_detailed_size-examples)
+Get size information for a hypertable.
 ```sql
-SELECT table_bytes, index_bytes, toast_bytes, total_bytes
-FROM hypertable_relation_size('conditions');
-```
-The expected output:
-```
- table_bytes | index_bytes | toast_bytes | total_bytes
--------------+-------------+-------------+-------------
-  1227661312 |  1685979136 |      180224 |  2913820672
+-- disttable is a distributed hypertable --
+SELECT * FROM hypertable_detailed_size('disttable') ORDER BY node_name;
+
+ table_bytes | index_bytes | toast_bytes | total_bytes |  node_name
+-------------+-------------+-------------+-------------+-------------
+       16384 |       32768 |           0 |       49152 | data_node_1
+        8192 |       16384 |           0 |       24576 | data_node_2
+
 ```
 ---
 
-## hypertable_relation_size_pretty() [](hypertable_relation_size_pretty)
+## chunks_detailed_size()  :community_function: [](chunks_detailed_size)
 
-Get relation size of hypertable like `pg_relation_size(hypertable)`.
+Get per chunk size information of hypertable.
+This function will return multiple rows of output for a single
+chunk, if this is a distributed hypertable. Meta data for a chunk can be
+viewed using `timescaledb_information.chunks` view.
 
-#### Required Arguments [](hypertable_relation_size_pretty-required-arguments)
+#### Required Arguments [](chunks_detailed_size-required-arguments)
 
 |Name|Description|
 |---|---|
-| `main_table` | Identifier of hypertable to get relation size for.|
+| `main_table` | Name of the hypertable|
```

Comment:
Given the description, this only works for hypertables, and not regular tables, right?

Why isn't the name of the argument "hypertable"?